src/components/Login.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+// comentario temporal
+
+export const Login = () => {
+    const [Email,setEmail]=useState("");
```

Comment:
const [email, setEmail] = useState("");

The first variable should be lowercase. Please review the React useState documentation: https://react.dev/reference/react/useState#usestate